### PR TITLE
Meta: update repository files

### PR DESCRIPTION
**Pull request by foolip**
See https://github.com/whatwg/spec-factory for details.

### CODE REVIEW

**Comment by foolip** [comment]
https://github.com/whatwg/console/pull/173 hasn't landed yet so this includes changes from that PR too. Would it be OK if I just back out those changes manually here to get the other stuff done?

**Comment by annevk** [comment]
@foolip it's fine to exclude the editorconfig changes and squash the result, yes.

**Comment by foolip** [comment]
This doesn't include .editorconfig now, after https://github.com/whatwg/spec-factory/pull/17.

File: Makefile [hunk 1]
```diff
@@ -2,10 +2,21 @@ SHELL=/bin/bash -o pipefail
...
+	                       -F md-Text-Macro="COMMIT-SHA LOCAL COPY" \
+	                       -F file=@index.bs) && \
+	[[ "$$HTTP_STATUS" -eq "200" ]]) || ( \
+		echo ""; cat index.html; echo ""; \
+		rm -f index.html; \
+		exit 22 \
+	);
 
 local: index.bs
-	bikeshed spec index.bs index.html --md-Text-Macro="SNAPSHOT-LINK LOCAL COPY"
```
Comment (foolip) [inline Makefile:8]: Note that `make local` was broken before this.

**Comment by foolip** [comment]
Success: https://github.com/whatwg/console/runs/769603255 / https://console.spec.whatwg.org/

**Pull request merged**